common/src/main/java/io/netty/util/concurrent/DefaultPromise.java

```diff
@@ -16,6 +16,7 @@
 package io.netty.util.concurrent;
```

Comment:
Can you tell me a bit more about these changes ?

Reply:
Oh, these are the remains of the changes discussed in https://github.com/netty/netty/pull/14358#discussion_r1774190996
They should be reverted.